src/components/Button/ButtonLink/index.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { LayoutType } from '../ButtonContent'
+import { ButtonLink } from '.'
+import renderWithTheme from '../../../testing/renderWithTheme'
+import { Switch, Route } from 'react-router-dom'
+import { BrowserRouter as Router } from 'react-router-dom'
+
+test('renders button with link and layout center', () => {
```

Comment:
This is duplicate of previous button test. Here it should test that page URL is changed by routing when user clicks on a button. :smile: